pkg/vault/test/authorizer/TimelockAuthorizerPermissions.test.ts

```diff
@@ -665,6 +598,48 @@ describe('TimelockAuthorizer permissions', () => {
             });
           });
         });
+
+        context('when the user has the permission in a specific contract and everywhere', () => {
```

Comment:
```suggestion
        context('when the user has both global permission and permission in a specific contract', () => {
```

Reply:
While this is better, I'm keeping (here and elsewhere) the original phrasing as it matches the rest of the suite. I'd rather it be sort of ugly but consistent, inconsistency will only make this even harder to read.